pom.xml

```diff
@@ -809,7 +821,7 @@
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
         <identity.governance.version>1.11.11</identity.governance.version>
-        <carbon.identity.framework.version>7.6.9</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.7.4-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
This version will be updated as https://github.com/wso2/carbon-identity-framework/pull/6148 is merged